R/mod_uploadPackage.R

```diff
@@ -127,15 +127,13 @@ uploadPackageServer <- function(id, user, auto_list, credentials, parent) {
         if (nrow(uploaded_pkgs()) > 0) upload_pkg_comp
       )
     })
-
-    cran_pkgs <- reactiveVal()
     
     observeEvent(input$load_cran, {
```

Comment:
should this input be changed to something more descriptive at this junction? Perhaps maintaining consistency with other recent changes in this PR: `input$load_repo_pkgs`? Note this change would impact a few areas of the code base:

![image](https://github.com/pharmaR/riskassessment/assets/17878953/8d474450-9642-4e8a-b71e-bc8a43d0aa22)